packages/concerto-core/lib/introspect/illegalmodelexception.js

```diff
@@ -43,7 +44,7 @@ class IllegalModelException extends BaseFileException {
      * @param {number} fileLocation.end.column - end column of the error location.
      * @param {string} [component] - the component which throws this error
      */
-    constructor(message, modelFile, fileLocation, component) {
+    constructor(message, modelFile, fileLocation, component, errorType = ErrorCodes.DEFAULT_VALIDATOR_EXCEPTION) {
```

Comment:
You can add a new exception code for `IllegalModelException`

Reply:
> You can add a new exception code for `IllegalModelException`

Yes! Added that